bin/mail

```diff
@@ -1037,6 +1035,11 @@ sub Batch {
 	$mailer->send($message);
 }
 
+sub VERSION_MESSAGE {
+	print "$0 version $VERSION\n";
```

Comment:
Let's fiddle with `$0` to get the basename for the program like we've done in other programs.

Reply:
Ok, I've added $Program here variable for consistency